src/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.csproj

```diff
@@ -53,6 +53,31 @@
     <Compile Include="System\Runtime\InteropServices\MarshalAsAttributeTests.cs" />
     <Compile Include="System\Runtime\InteropServices\MarshalTests.cs" />
     <Compile Include="System\Runtime\InteropServices\MarshalTests.Windows.cs" Condition="'$(TargetsWindows)' == 'true'" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\CommonHelper.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\ByteArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\CharArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\DoubleArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\Int16ArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\Int32ArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\Int64ArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\IntPtrArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\Copy\SingleArrayTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\GetExceptionForHRTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\GetNativeVariantForObjectTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\GetObjectForNativeVariantTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\GetObjectsForNativeVariantsTests.cs" />
```

Comment:
Currently These testcase should be Windows only.

Reply:
If the tests are Windows specific, they should be excluded using  `[PlatformSpecific(TestPlatforms.Windows)]` attribute in the sources.